ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml

```diff
@@ -43,7 +43,7 @@ tests:
     cluster_profile: aws-2
     env:
       HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.12.19-multi
-      HYPERSHIFT_NODE_COUNT: "6"
+      HYPERSHIFT_NODE_COUNT: "12"
```

Comment:
Staging has 9 nodes (3 masters, 6 workers), production has 12 nodes (3 masters, 9 workers), I don't think that our testing env should have more workers than staging.